bin/lib/amazon.py

```diff
@@ -219,6 +219,11 @@ def release_for(releases, s3_key):
     return None
 
 
+def get_current_release(cfg: Config) -> Optional[Release]:
```

Comment:
Thanks for type annotating. I do a lot of Python these days and I'm overdue tidying up this codebase...